website_sale_template_adj/readme/CONTRIBUTORS.rst

```diff
@@ -0,0 +1 @@
+* Quartile Limited <info@quartile.co>
```

Comment:
No need to have this file for our custom modules.

Reply:
I got it!